js/field-gmap.js

```diff
@@ -3,15 +3,16 @@
 
 (function($) {
 
-	if ( 'undefined' == typeof google ) {
-		$('.map').css('textAlign','center').append( 'Google Maps API not loaded.' );
-		return;
-	}
-
 	var CMBGmapsInit = function( fieldEl ) {
 
+		var mapCanvas = $('.map', fieldEl ).get(0);
+
+		if ( 'undefined' == typeof google ) {
+			$('<div>'+CMBGmaps.strings.googleMapsApiNotLoaded+'</div>').css({'width':'100%','textAlign':'center','marginTop':'1em'}).appendTo( mapCanvas );
```

Comment:
Could you adjust the spacing here? Need spaces around the strings, concatenation `+`s, etc.